third_party/icu/googleurl/googleurl_fuzz_test.cc

```diff
@@ -0,0 +1,16 @@
+#include "url/gurl.h"
+
+#include "test/fuzz/fuzz_runner.h"
+
+namespace Envoy {
+namespace Fuzz {
+namespace {
+
+DEFINE_FUZZER(const uint8_t* buf, size_t len) {
+  const std::string input(reinterpret_cast<const char*>(buf), len);
+  GURL url(input);
```

Comment:
Would it make sense to add some accessors on the GURL to make sure that lazily computed attributes don't crash?

Reply:
Will add it.